EpiserverRedirects/Import/RedirectsImporter.cs

```diff
@@ -32,13 +42,36 @@ private IRedirectRule CreateRedirectRule(RedirectRuleImportRow redirectRow)
                     Parser.ParseRedirectRuleType(redirectRow.RedirectRuleType),
                     Parser.ParseBoolean(redirectRow.IsActive),
                     redirectRow.Notes,
-                    redirectRow.Priority ?? _options.DefaultRedirectRulePriority)
+                    redirectRow.Priority ?? _options.DefaultRedirectRulePriority, DetermineHostId(redirectRow.Host))
                 : RedirectRuleModel.NewFromImport(redirectRow.OldPattern, redirectRow.ContentId.Value,
                     Parser.ParseRedirectType(redirectRow.RedirectType),
                     Parser.ParseRedirectRuleType(redirectRow.RedirectRuleType),
                     Parser.ParseBoolean(redirectRow.IsActive),
                     redirectRow.Notes,
-                    redirectRow.Priority ?? _options.DefaultRedirectRulePriority);
+                    redirectRow.Priority ?? _options.DefaultRedirectRulePriority, DetermineHostId(redirectRow.Host));
+        }
+
+        private Guid? DetermineHostId(string hostIdOrHostName)
+        {
+            if (string.IsNullOrEmpty(hostIdOrHostName))
+            {
+                return null;
+            }
+            if (Guid.TryParse(hostIdOrHostName, out var guid))
+            {
+                if (AllHosts.Where(s => s.Id == guid).IsNullOrEmpty())
+                {
+                    return null;
+                }
+
+                return guid;
+            }
+            var hosts = AllHosts.Where(s => s.Name == hostIdOrHostName).ToList();
```

Comment:
maybe it's better to use `s.Name.Equals(hostIdOrHostName, StringComparison.InvariantCultureIgnoreCase)` instead of `s.Name == hostIdOrHostName` (so we're comparing and ignoring case)?